tools/setup.py

```diff
@@ -53,10 +53,19 @@ def check_directory(self, name):
             self.color.print_pink(name + " directory found")
         else:
             self.color.print_red(
-                name + " directory not found, cloning for you")
-            subprocess.check_call("git clone "
-                                  + "https://github.com/SCE-Development/"
-                                  + name, stderr=subprocess.STDOUT, shell=True)
+                name + ' directory not found')
+            custom_dir = prompt_user(f'would you like to link an existing clone of {name}'
+                    ' to the sce cli?(y or n): ', lambda ans: ans == 'y' or ans == 'n')
+            if custom_dir == 'y':
+                user_project_dir = prompt_user(f'enter a valid, absolute path of {name}: ',
+                        lambda path: os.path.isdir(path) and os.path.isabs(path))
```

Comment:
instead of entering an absolute path could we add a new command like

```
sce link <project>
```

the user would run it in a project directory, wherever it is. then they get prompted with
```
would you like to use <path> as the location of <project> [yN]?
```

then you run the below `os.symlink`.

to get the location of the dev tool (assuming user did the setup already), its in `os.environ["SCE_PATH"]`

Reply:
Good idea. We can do this once #40 gets through. 

`SceSetupTool` requires a path to the respective projects to complete setup so we can link here as well as in the new subcommand.